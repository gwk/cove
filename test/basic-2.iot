{
  'cmd': 'cove test/basic.py 2',
  'links': 'test',
}
