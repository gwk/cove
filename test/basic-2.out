
cove report:

__main__: test/basic.py: 8 lines covered.
